src/stores/widgets/WidgetLayoutStore.ts

```diff
@@ -452,6 +456,11 @@ export class WidgetLayoutStore extends ReadyWatchingStore {
                 for (const w of this.getContainerWidgets(room, Container.Center)) {
                     this.moveToContainer(room, w, Container.Right);
                 }
+                // Additionally change the right panel phase to chat:
+                dis.dispatch<SetRightPanelPhasePayload>({
+                    action: Action.SetRightPanelPhase,
+                    phase: RightPanelPhases.TimelineCard,
+                });
```

Comment:
Should we put this inside one of the `onWidgetLayoutChange` callbacks to keep ui logic out of the store? Or is this fine, since its just a dispatch? (cannot fail in other environments like the WidgetLayoutStore-test)